lnrpc/rpc.proto

```diff
@@ -1152,8 +1152,10 @@ message Hop {
     uint64 chan_id = 1 [json_name = "chan_id"];
     int64 chan_capacity = 2 [json_name = "chan_capacity"];
     int64 amt_to_forward = 3 [json_name = "amt_to_forward"];
-    int64 fee = 4 [json_name = "fee"];
-    uint32 expiry = 5 [json_name = "expiry"];
+    int64 amt_to_forward_msat = 4 [json_name = "amt_to_forward_msat"];
+    int64 fee = 5 [json_name = "fee"];
+    int64 fee_msat = 6 [json_name = "fee_msat"];
+    uint32 expiry = 7 [json_name = "expiry"];
```

Comment:
In order to not break the set of existing protos, all these fields need to be added at the _end_.